src/kernel.jl

```diff
@@ -509,6 +509,26 @@ moffat(α::Real, β::Real)                 = moffat(α, β, ceil(Int, (α*2*sqrt
     sum(x.^2)
 end
 
+"""
+butterworth(n,Wn,ls{tuple}) -> k
+Returns a multidimensional dimensional Butterworth kernel contained in an OffsetArray(::Matrix{Float64})
+
+    - `n` is the order of the filter
+    - `Wn` is the normalized cutoff frequency
+    - `ls` is the size of the kernel
```

Comment:
You don't need indentation for the markdown list

```suggestion
- `n` is the order of the filter
- `Wn` is the normalized cutoff frequency
- `ls` is the size of the kernel
```